modules/ROOT/pages/clauses/index.adoc

```diff
@@ -12,13 +12,13 @@ This section contains information on all the clauses in the Cypher query languag
 [[administration-clauses]]
 == Administration clauses
 
-These comprise clauses used to manage databases, schema and security; further details can found in xref::databases.adoc[Database management] and xref::access-control/index.adoc[Access control].
+These comprise clauses used to manage databases, schema and security; further details can found in xref::administration/databases.adoc[Database management] and xref::administration/access-control/index.adoc[Access control].
```

Comment:
Maybe not part of the restructure as such but should this section also get the alias and server commands?

Reply:
Sounds reasonable (but not in this PR) :bowtie: